pkg/apis/work/v1alpha2/binding_types_helper.go

```diff
@@ -57,6 +57,14 @@ func (s *ResourceBindingSpec) GracefulEvictCluster(name, producer, reason, messa
 		return
 	}
 
+	s.Clusters = append(s.Clusters[:i], s.Clusters[i+1:]...)
+
+	for _, task := range s.GracefulEvictionTasks {
+		if task.FromCluster == s.Clusters[i].Name {
+			return
```

Comment:
Do we need to update the eviction information?

Reply:
I don't think it's necessary to update the new information if the older version of the task has not been completed.